packages/modular-scripts/src/utils/getPackageDependencies.ts

```diff
@@ -112,20 +114,24 @@ function parseYarnLockV1(
   lockFile: string,
   deps: Dependency,
 ): Dependency | null {
+  let parsedLockfile: ReturnType<typeof lockfile.parse>;
   try {
-    const parsedLockfile = lockfile.parse(lockFile);
-    return Object.entries(deps).reduce<Record<string, string>>(
-      (acc, [name, version]) => {
-        acc[name] = (parsedLockfile.object as LockFileEntries)[
-          `${name}@${version}`
-        ].version;
-        return acc;
-      },
-      {},
-    );
+    parsedLockfile = lockfile.parse(lockFile);
```

Comment:
This is a bit confusingly named - `lockile` and `lockFile`

Reply:
Yep - renamed it with a more descriptive / less confusing name.